xgi/convert/incidence.py

```diff
@@ -37,7 +37,7 @@ def from_incidence_matrix(d, create_using=None, nodelabels=None, edgelabels=None
 
     See Also
     --------
-    incidence_matrix
+    ~xgi.linalg.hypergraph_matrix.incidence_matrix
     to_incidence_matrix
```

Comment:
how come the first line needs the long reference and the second one does not?

Reply:
You need the long reference if it's not in the same file.